src/main/java/clients/model/TestTypes.java

```diff
@@ -1,15 +1,18 @@
 package clients.model;
 
 public enum TestTypes {
-    AAV2("40", "aav2", TestTypeClassification.CERTIFICATE.getValue()),
-    AAV3("40", "aav3", TestTypeClassification.CERTIFICATE.getValue()),
-    AAV4("40", "aav4", TestTypeClassification.CERTIFICATE.getValue()),
-    AAV5("40", "aav5", TestTypeClassification.CERTIFICATE.getValue()),
-    AAT1("94", "aat1", TestTypeClassification.CERTIFICATE.getValue()),
-    AAT2("40", "aat2", TestTypeClassification.CERTIFICATE.getValue()),
-    AAT3("40", "aat3", TestTypeClassification.CERTIFICATE.getValue()),
-    AAT4("40", "aat4", TestTypeClassification.CERTIFICATE.getValue()),
-    AAT5("40", "aat5", TestTypeClassification.CERTIFICATE.getValue()),
+    AAV2("94", "aav2", TestTypeClassification.CERTIFICATE.getValue()),
+    AAV3("94", "aav3", TestTypeClassification.CERTIFICATE.getValue()),
+    AAV4("94", "aav4", TestTypeClassification.CERTIFICATE.getValue()),
+    AAV5("94", "aav5", TestTypeClassification.CERTIFICATE.getValue()),
+    AAT1("40", "aat1", TestTypeClassification.CERTIFICATE.getValue()),
+    AAT2("94", "aat2", TestTypeClassification.CERTIFICATE.getValue()),
+    AAT3("94", "aat3", TestTypeClassification.CERTIFICATE.getValue()),
+    AAT4("94", "aat4", TestTypeClassification.CERTIFICATE.getValue()),
+    AAT5("94", "aat5", TestTypeClassification.CERTIFICATE.getValue()),
+//    TGV2("40", "tgv2", TestTypeClassification.CERTIFICATE.getValue()),
+//    TGV3("40", "tgv3", TestTypeClassification.CERTIFICATE.getValue()),
+//    TGV4("40", "tgv4", TestTypeClassification.CERTIFICATE.getValue()),
```

Comment:
Why are these items marked as commented out?

Reply:
Because they are added in the excel files, but there are not yet present in the json file used for populating the test-types dynamo db table.

Reply:
Ah yes, I see your comment about why they are disabled.